### ISSUE TITLE
Add New button disappears

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/swoptact#214]
When adding a new contact to an institution, the add new button disappears under the following conditions:

- create a new (non-existent in DB) participant
- tab to next cell

OR:
- add a new, not in DB, participant and hit enter

OR: 
- add a new contact that already existed in DB (from autocomplete) and hit enter

So, basically any time a new contact is saved, the add new button disappears.  I'm not sure when this turned up, but it is a bug.  The add new button will reappear on refresh of the page (up to 4 contacts) but it shouldn't disappear at all.

**Comment by cecilia-donnelly** [comment]
I talked to @nttaylor about this and he said [some of the following is verbatim from IRC, but it's been heavily edited for length and clarity]:

The problem is in the `"checkLimitReached"` function.  Basically it's deciding the limit has been reached after any commit, and so it takes the add new button away.  There's a line that's like `"linkedModels = $("tr")"` which counts the total number of `<tr>` elements in the whole document!  So of course we'll be over the limit every time!  So in my working copy I changed it to `"$("#inlined-model-table tbody tr")` to count only those trs in the inline table (and not the headers either).

There are two other problems
 1. When you "enter" or "tab" and thus create a new row, this happens before "checkLimit" is called, so it will count a blank row with nothing in it before making the "add new" button go away.  The "delete row" button on this new row doesn't work.
2.   I guess there is an element called "inlined-error-box" where the message that says "you've reached the limit" is supposed to appear.  I didn't look in the "inline_models" template but it does appear to be there, but the error message isn't appearing.

Also I don't think the add new button should just disappear. Better that it be deactivated and the error message would appear and reappear when the user clicked on the deactivated button
